### ISSUE TITLE
Not a problem but a question -- Color coding parameters? How to change?

### ISSUE THREAD

**Issue by Watson-79** [schmich/marinara#261]
Hi, I have a question re. color coding. 

The color coding is a great feature, I find it also great that the parameters change depending on use (eg. very dark olive green represents the most productive segment and it is automatically adjusted depending on the data). My question: is it possible to set the variables manually? For example: every day over 10 pomodoros will be marked automatically with the darkest color?  

Thanks!

**Comment by schmich** [comment]
No, right now, as you've noticed, the classification is dynamic based on your activity. It currently cannot be configured.

If you'd like, we can leave this issue open to track this as a feature request.

**Comment by Watson-79** [comment]
Hi, that would be great.

I understand the logic behind this feature, but I think at a certain point
it stops encouraging productivity. To think this through to the last
consequence: as the system does not take extreme datapoints into account
(outliers), it ultimately generates a a dynamic that ends in a 24 hrs
workday :) A feature that allowed for setting variables would solve that.
Thanks!
